tools/cli/commands/rsync.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
```

Comment:
Some of my suggestions below need the "path" module. Making a suggestion since that can make it easier to do things in GH's UI.
```suggestion
import fs from 'fs';
import path from 'path';
```